Makefile

```diff
@@ -0,0 +1,3 @@
+all:
+	javac -d bin src/calculator/Calculator.java; \
+	java -classpath ./bin/ calculator/Calculator
```

Comment:
this just allows me to run `make` in the terminal window and have the program run